kafka_consumer/datadog_checks/kafka_consumer/client.py

```diff
@@ -6,11 +6,10 @@
 from six import string_types
 
 from datadog_checks.base import ConfigurationError
-from datadog_checks.kafka_consumer.client.kafka_client import KafkaClient
 from datadog_checks.kafka_consumer.constants import KAFKA_INTERNAL_TOPICS
 
 
-class ConfluentKafkaClient(KafkaClient):
```

Comment:
Although I doubt we would switch to a new library in the near future, can we still keep the `ConfluentKafkaClient` + `KafkaClient` inheritance?

Reply:
No need to keep something that we are almost 100% sure we won't need. Let's keep the code as simple as possible.  Basically YAGNI.

If for something we decide to use another library in the future, it won't be too difficult to do something along those lines.